cookbooks/particle_injection.prm

```diff
@@ -0,0 +1,247 @@
+# At the top, we define the number of space dimensions we would like to
```

Comment:
Please go through the file and remove unnecessary comment lines. Also add a small sentence at the beginning what this test is about. Not super important, but nicer if somebody needs to look into this test in the future.